docs/workflow/tutorial-web-qr-code-activity.mdx

```diff
@@ -0,0 +1,184 @@
+
+## Test the Activity
+
+Once your activity pack is hosted and registered, your custom activity should appear in the activity toolbox in Geocortex Workflow Designer alongside the built-in activities, and can be used in the graphical interface like any other activity.
+
+:::tip
+The `result` output of the activity
+:::
+
+Now you can build a workflow that uses your new custom activity!
+
+## Enhance the Activity
+
+Once you've seen your activity in action you may want to take advantage of the many different options offered by the [qrcode](https://github.com/soldair/node-qrcode) library. To do this you can add additional inputs to the activity that correspond to options on the `QRCode.toDataURL(text: string, options?: QRCodeToDataURLOptions)` method.
+
+Modify the previous activity implementation in `src/activities/CreateQrCodeImage.ts` to match the following more complete example.
+
+```tsx title="src/activities/CreateQrCodeImage.ts"
+import type { IActivityHandler } from "@geocortex/workflow/runtime/IActivityHandler";
+import QRCode, { QRCodeToDataURLOptions } from "qrcode";
+
+/** An interface that defines the inputs of the activity. */
+export interface CreateQrCodeImageInputs {
+    // @description The text or data to encode into the QR code.
+    // @required
+    text:
+        | string
+        | {
+              data: string;
+              mode: "alphanumeric" | "numeric" | "kanji" | "byte";
+          }[];
+
+    // @description The RGBA color of the dark module. The default is #000000ff (black).
+    darkColor?: string;
+
+    // @description Error correction level. The default is medium.
+    errorCorrectionLevel?: "low" | "medium" | "quartile" | "high";
+
+    // @description The RGBA color of the light module. The default is #ffffffff (white).
+    lightColor?: string;
+
+    // @description The size of the quiet zone around the QR code. The default is 4.
+    margin?: number;
+
+    // @description A Number between 0 and 1 indicating image quality if the requested type is image/jpeg or image/webp. The default is 0.92.
+    quality?: number;
+
+    // @description The Scale factor. A value of 1 means 1px per modules (black dots). The default is 4.
+    scale?: number;
+
+    // @description The image format. The default is image/png.
+    type?: "image/png" | "image/jpeg" | "image/webp";
+
+    // @description The width of the output image. If the width is too small to contain the QR code, this option will be ignored. This input takes precedence over the scale input.
+    width?: number;
+
+    // @description The QR Code version. A number between 1 and 40. If not specified, a suitable value will be calculated.
+    version?: number;
+}
+
+/** An interface that defines the outputs of the activity. */
+export interface CreateQrCodeImageOutputs {
+    /** The data URL of the resulting the QR code image. */
+    // @description The data URL of the resulting the QR code image.
```

Comment:
Do we know if the @description is picked up by our tooling if used in the JSDoc above?

Reply:
Good catch. WF Designer does use the @description values for SDK activities. The JSDoc comments are only used by the built-in product activities. I've removed it.

Reply:
I've changed this file to use JSDoc comments. I'll roll this through all the other examples shortly.

Reply:
I've applied JSDoc comments to all the pages.